cloud_service/acscs-recommended-requirements.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="acscs-recommended-requirements"]
+= Recommended resource requirements for {product-title-managed}
+include::modules/common-attributes.adoc[]
+:context: acscs-recommended-requirements
+
+toc::[]
+
+Recommended resource requirements provide additional details about required resources based on workload size on secured clusters.
+Used resources depend on workload type and size on secured clusters. It's difficult to take one Kubernetes resource and define a sizing guide based on that. Please take into consideration that the following size recommendations are based on focused testing on a single secured cluster.
+
+This test creates the following objects across a given number of namespaces:
```

Comment:
This part and the part after the bullets is confusing to me, because it talks about "this test" but there is no information about testing...is this referring to the testing that was done in order to come up with these guidelines? If so, may want to just say something general like "The recommended resource guidelines were developed by performing a test that created the following objects across a given number of namespaces:"

Reply:
Although honestly this is more than the usual level of detail that I've seen given for recommended guidelines, but if you feel that it is important for the user to know this, I defer to your recommendation. Normally we would just say something like "based on our testing, we recommend x."

Reply:
I have applied your suggestion. I used the word "focused" test.

Inspiration for this page comes from OpenShift: https://docs.openshift.com/container-platform/4.13/scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.html#master-node-sizing_recommended-control-plane-practices